ironfish/src/network/messages/newBlock.test.ts

```diff
@@ -1,70 +1,59 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
-import { SerializedBlock } from '../../primitives/block'
+import { Block, BlockHeader, Target } from '../../primitives'
+import { createNodeTest, useTxSpendsFixture } from '../../testUtilities'
 import { NewBlockMessage } from './newBlock'
 
 describe('NewBlocksMessage', () => {
-  it('serializes the object into a buffer and deserializes to the original object', () => {
+  const nodeTest = createNodeTest()
+
+  function expectNewBlockMessageToMatch(a: NewBlockMessage, b: NewBlockMessage): void {
```

Comment:
Same with this function, can we reuse [`Block.equals()`](https://github.com/iron-fish/ironfish/blob/67da183ca994987db05e3b061dec6768f8859f3c/ironfish/src/primitives/block.ts#L70)

Reply:
Changed to reuse `Block.equals()`